DangerSwap/DangerSwap/Services/Currency/CurrencyService.cs

```diff
@@ -57,6 +58,7 @@ private async Task UpdateCurrency(Currency originCurrency, ScrappedCurrency scra
         private void OverrideCurrencyRate(Currency originCurrency, ScrappedCurrency scrappedCurrency)
```

Comment:
Method "OverrideCurrencyRate" doesn't access instance data and can be made static